src/PKSim.Core/CoreConstants.cs

```diff
@@ -252,6 +259,7 @@ public static class CalculationMethod
          public static readonly string LinksCommon = "LinksCommon";
          public static readonly string DistributionInVitroPKSim = "DistributionInVitro_PKSim";
          public static readonly string ApplicationParameter = "ApplicationParameter";
+         public static readonly string ApplicationParameter_Human = "ApplicationParameter_Human";
```

Comment:
no Category (and CM). You can only have one active CM per category. ApplicaitonParameter is still used for Human. The `ApplicationParameter_human` is used as an extra category